src/components/Section/Foreign/Business/FamilyItem.test.jsx

```diff
@@ -14,8 +14,9 @@ describe('The family item component', () => {
     const component = mount(<FamilyItem {...expected} />)
     component.find('.family-name .first input').simulate('change')
     component.find('.family-agency input').simulate('change')
+    component.find('.family-country .react-autosuggest__container input').simulate('change', { target: { value: 'C' } })
```

Comment:
@mgwalker have you seen much use of something similar to test helpers with unit testing in ReactJS?

For example, something like:

```jsx
simulateForCountry('.family-country', 'change', { targe: { value: 'C' } })
```

Reply:
To be honest, the vast majority of my experience with React hasn't included tests at all.  😬